tracer/src/Datadog.Trace.Tools.Runner/Checks/Resources.cs

```diff
@@ -70,27 +70,15 @@ internal static class Resources
 
         public static string WrongProfilerRegistry(string registryKey, string actualProfiler) => $"The registry key {registryKey} was set to '{actualProfiler}' but it should point to 'Datadog.Trace.ClrProfiler.Native.dll'. Please check that all external profilers have been uninstalled properly and try reinstalling the tracer.";
 
-        public static string CouldNotFindSite(string site, IEnumerable<string> availableSites)
-        {
-            var sb = new StringBuilder();
-
-            sb.AppendLine($"Could not find site {site}");
-            sb.AppendLine("Available sites:");
-
-            foreach (var s in availableSites)
-            {
-                sb.AppendLine($" - {s}");
-            }
+        public static string IisApplicationNotProvided() => "IIS application name not provided. ";
 
-            return sb.ToString();
-        }
+        public static string CouldNotFindIisApplication(string site, string application) => $"Could not find IIS application \"{site}{application}\". ";
 
-        public static string CouldNotFindApplication(string site, string application, IEnumerable<string> availableApplications)
+        public static string ListAllIisApplications(IEnumerable<string> availableApplications)
         {
             var sb = new StringBuilder();
 
-            sb.AppendLine($"Could not find application {application} in site {site}");
-            sb.AppendLine("Available applications:");
+            sb.AppendLine("Available IIS applications:");
```

Comment:
We should add text here on how to use this list. Probably a simple version of the default usage text:
```
> dd-trace check iis -h

USAGE:
    dd-trace check iis [siteName] [OPTIONS]

ARGUMENTS:
    [siteName]

OPTIONS:
    -h, --help    Prints help information
```

Reply:
added below in d47f4355c83eed3c7160eaf66ca850f4881ed335